packages/block-editor/src/components/inserter/menu.native.js

```diff
@@ -83,38 +86,40 @@ export class InserterMenu extends Component {
 				contentStyle={ [ styles.content, bottomPadding ] }
 				hideHeader
 			>
-				<FlatList
-					onLayout={ this.onLayout }
-					scrollEnabled={ false }
-					key={ `InserterUI-${ this.state.numberOfColumns }` } //re-render when numberOfColumns changes
-					keyboardShouldPersistTaps="always"
-					numColumns={ this.state.numberOfColumns }
-					data={ this.props.items }
-					ItemSeparatorComponent={ () => <View style={ styles.rowSeparator } /> }
-					keyExtractor={ ( item ) => item.name }
-					renderItem={ ( { item } ) => (
-						<TouchableHighlight
-							style={ styles.touchableArea }
-							underlayColor="transparent"
-							activeOpacity={ 0.5 }
-							accessibilityLabel={ item.title }
-							onPress={ () => this.props.onSelect( item ) }
-						>
-							<View style={ styles.modalItem }>
-								<View style={ modalIconWrapperStyle }>
-									<View style={ modalIconStyle }>
-										<Icon
-											icon={ item.icon.src }
-											fill={ modalIconStyle.fill }
-											size={ modalIconStyle.width }
-										/>
+				<TouchableHighlight>
```

Comment:
It's a bit hard to explain to will use a gif to illustrate it better:

* without `Touchable` or with a `View`: if you will drag an empty space within a bottom-sheet instead of scrolling you can swipe to close the bottom sheet

![without_toucable](https://user-images.githubusercontent.com/22746080/73655906-50e9f900-468f-11ea-9d17-22e977807c42.gif)

* with `Touchable`: dragging an empty space is scrolling instead of swiping

![with_touchable](https://user-images.githubusercontent.com/22746080/73656061-a1f9ed00-468f-11ea-9d00-91e730c1831c.gif)


Let me know if something is unclear with that!

Reply:
Perfect! Thanks for the detailed explanation.